nim/app.py

```diff
@@ -34,12 +33,16 @@ def update_game_state():
 
 @app.route('/new', methods=['GET', 'POST'])
 def new_game():
-    json = loads('{"min": 3, "max": 12, "piles": 3}')
-    state = app.game.new_game(
-        min=json['min'],
-        max=json['max'],
-        piles=json['piles'],
-    )
+    """
+    route for /new so user can start new game. Uses JSON from frontend
+    for params in new game
+    """
+    user_request = request.get_json()
+    game_request = {**user_request}
+    try:
+        state = app.game.new_game(**game_request)
```

Comment:
At least for now, I think we could we could simplify this:
```
        state = app.game.new_game(**request.get_json())
```

Reply:
Oh shit, dat nice.